lib/suspenders/app_builder.rb

```diff
@@ -120,9 +120,11 @@ def setup_asset_host
         "# config.action_controller.asset_host = 'http://assets.example.com'",
         'config.action_controller.asset_host = ENV.fetch("ASSET_HOST", ENV.fetch("APPLICATION_HOST"))'
 
-      replace_in_file 'config/initializers/assets.rb',
-        "config.assets.version = '1.0'",
-        'config.assets.version = (ENV["ASSETS_VERSION"] || "1.0")'
+      if File.exist?("config/initializers/assets.rb")
+        replace_in_file 'config/initializers/assets.rb',
```

Comment:
I think an even better solution to this problem would be adding the `if File.exist?(path)` check in `replace_in_file`, which would also fix the errors that occur when running `suspenders --skip-action-mailer`.